HTML/HTML.sublime-syntax

```diff
@@ -10,9 +10,9 @@ first_line_match: (?i)<(!DOCTYPE\s*)?html
 scope: text.html.basic
 
 variables:
-  attribute_char: (?:[^ "'>/=\x00-\x1f\x7f-\x9f])
+  # https://html.spec.whatwg.org/multipage/syntax.html#attributes-2
+  attribute_char: '[^\t\n\f />=]'
   unquoted_attribute_value: (?:[^\s<>/''"]|/(?!>))+
```

Comment:
@Thom1729 What do you think about `\s` here? I wonder, if it should be replaced by `\t\n\f ` as well.